src/pages/[platform]/build-a-backend/storage/upload-files/index.mdx

```diff
@@ -68,10 +66,6 @@ Learn more about how you can further customize the UI component by referring to
 
 ## API Usage
```

Comment:
I would still retitle this header - Implement Uploads for ex.

Reply:
Changed to `Implement upload functionality`